algebras/algebra/src/main/scala/endpoints/algebra/Endpoints.scala

```diff
@@ -43,16 +43,43 @@ trait EndpointsWithCustomErrors extends Requests with Responses with Errors {
     *
     * @param request  Request
     * @param response Response
-    * @param summary optional summary documentation
-    * @param description optional description documentation
-    * @param tags list of OpenApi tags
+    * @param docs     Documentation (used by documentation interpreters)
     */
   def endpoint[A, B](
     request: Request[A],
     response: Response[B],
+    docs: EndpointDocumentation = EndpointDocumentation()
+  ): Endpoint[A, B]
+
+  /**
+    * @param summary     Short description
+    * @param description Detailed description
+    * @param tags        OpenAPI tags
+    * @param callbacks   Callbacks indexed by event name
+    */
+  case class EndpointDocumentation(
```

Comment:
Maybe we should name this type `EndpointDocs`, for conciseness and consistency with the `docs` parameter name of the `endpoint` operation.